docs/swarm/manage-nodes.md

```diff
@@ -0,0 +1,221 @@
+* [add or remove label metadata](#add-or-remove-label-metadata)
+* [change a node role](#promote-or-demote-a-node)
+
+### Change node availability
+
+Changing node availability lets you:
+
+* drain a manager node so that only performs swarm management tasks and is
+unavailable for task assignment.
+* drain a node so you can take it down for maintenance.
+* pause a node so it is unavailable to receive new tasks.
+* restore unavailable or paused nodes available status.
+
+For example, to change a manager node to `Drain` availability:
+
+```bash
+$ docker node update --availability drain node-1
+
+node-1
+```
+
+See [list nodes](#list-nodes) for descriptions of the different availability
+options.
+
+### Add or remove label metadata
+
+Node labels provide a flexible method of node organization. You can also use
+node labels in service constraints. Apply constraints when you create a service
+to limit the nodes where the scheduler assigns tasks for the service.
+
+Run `docker node update --label-add` on a manager node to add label metadata to
+a node. The `--label-add` flag supports either a `<key>` or a `<key>=<value>`
+pair.
+
+Pass the `--label-add` flag once for each node label you want to add:
+
+```bash
+$ docker node update --label-add foo --label-add bar=baz node-1
+
+node-1
+```
+
+The labels you set for nodes using docker node update apply only to the node
+entity within the swarm. Do not confuse them with the docker daemon labels for
+[dockerd](../userguide/labels-custom-metadata.md#daemon-labels).
+
+Refer to the `docker service create` [CLI reference](../reference/commandline/service_create.md)
+for more information about service constraints.
+
+### Promote or demote a node
+
+You can promote a worker node to the manager role. This is useful when a
+manager node becomes unavailable or if you want to take a manager offline for
+maintenance. Similarly, you can demote a manager node to the worker role.
+
+Regardless of your reason to promote or demote a node, you should always
+maintain an odd number of manager nodes in the swarm. For more information refer
+to the [Swarm administration guide](admin_guide.md).
+
+To promote a node or set of nodes, run `docker node promote` from a manager
+node:
+
+```bash
+$ docker node promote node-3 node-2
+
+Node node-3 promoted to a manager in the swarm.
+Node node-2 promoted to a manager in the swarm.
+```
+
+To demote a node or set of nodes, run `docker node demote` from a manager node:
+
+```bash
+$ docker node demote node-3 node-2
+
+Manager node-3 demoted in the swarm.
+Manager node-2 demoted in the swarm.
+```
+
+`docker node promote` and `docker node demote` are convenience commands for
+`docker node update --role manager` and `docker node update --role worker`
+respectively.
+
+
+## Remove a node from the swarm
+
+Before you can remove a node from the swarm, you must first run the
+`docker swarm leave` command on the none you want to remove.
```

Comment:
Actually, the sentence doesn't really make sense. `docker swarm leave` removes the node from the swarm, so it's not something you do before removing the node.